system-int-test/test/cmr/system_int_test/ingest/variable_ingest_test.clj

```diff
@@ -216,3 +218,39 @@
             (is (= 200 (:status update-response))))
           (testing "update variable status"
             (is (= 200 (:status delete-response)))))))))
+
+(defn- ingest-variable
+  "This is a temporary function created to ingest a variable with the specified metadata.
+   Once we move the variable's CRUD API onto provider routes in CMR-4239, we should remove
+   this function and update the invalid-variable-metadata-ingest-test to use
+   ingest-util/ingest-concept function instead."
+  [token metadata]
+   (let [params {:method :post
+                 :url (url/ingest-create-variable-url)
+                 :body  metadata
+                 :content-type "application/vnd.nasa.cmr.umm+json"
+                 :headers {"Echo-Token" token}
+                 :accept :xml
+                 :throw-exceptions false
+                 :connection-manager (s/conn-mgr)}]
+     (ingest-util/parse-ingest-response (client/request params) {})))
+
+(deftest invalid-variable-metadata-ingest-test
```

Comment:
I didn't add a test for variable update as we are moving variable ingest to providers routes and the create and update routes will be combined as in CMR-4239. I will add demos to show update also validates against JSON schema now.